projects/core/src/cart/store/actions/cart.action.spec.ts

```diff
@@ -113,6 +117,23 @@ describe('Cart Actions', () => {
     });
   });
 
+  describe('LoadCarts Actions', () => {
+    describe('LoadCartsSuccess', () => {
+      it('should create the action', () => {
+        const payload = [savedCart];
+        const action = new CartActions.LoadCartsSuccess(payload);
+        expect({ ...action }).toEqual({
+          type: CartActions.LOAD_CARTS_SUCCESS,
+          payload: [savedCart],
+          meta: StateUtils.entitySuccessMeta(
+            MULTI_CART_DATA,
+            payload.map((cart) => cart?.code)
```

Comment:
Could you in here just insert static array? This looks like copy paste from action implementation